cmd/fleet/serve.go

```diff
@@ -1145,7 +1145,8 @@ the way that the Fleet server works.
 					}
 				}
 
-				if req.Method == http.MethodPost && strings.HasSuffix(req.URL.Path, "/fleet/software/package") {
+				if (req.Method == http.MethodPost && strings.HasSuffix(req.URL.Path, "/fleet/software/package")) ||
+					(req.Method == http.MethodPatch && strings.HasSuffix(req.URL.Path, "/package") && strings.Contains(req.URL.Path, "/fleet/software/titles/")) {
```

Comment:
sanity-check: do you know if the UI uses the exact same component as well? the library we use to make requests has a default timeout to abort requests that's lower than we want. 

see https://github.com/fleetdm/fleet/pull/20479/files